dags/aps/aps_api_client.py

```diff
@@ -0,0 +1,43 @@
+from os import getenv
+import json
+
+from common.rest_api_client import RestApiClient
+
+
+class APSApiClient(RestApiClient):
+    def __init__(self):
+        super().__init__()
+        self.base_url = getenv("BASE_URL_FOR_APS", 'http://harvest.aps.org/v2/journals/articles')
+        self.rest_api_client = RestApiClient(base_url=self.base_url)
```

Comment:
There is some weird relationship between those classes here. APSApiClient `is` a RestApiClient (as it is the base class) and it also `has` a RestApiClient